src/test/lib/utils/lossless_predicate_cast_test.cpp

```diff
@@ -18,7 +18,7 @@ TEST_F(LosslessPredicateCastTest, NextFloatTowards) {
   // Edge cases:
   EXPECT_EQ(next_float_towards(3.1, 3.1), std::nullopt);
 
-  // // Maximum double that can losslessly represented as a float and the next higher double
+  // Maximum double that can losslessly represented as a float and the next higher double
```

Comment:
This has nothing to do with plugins or this PR but I found the double comment while searching for that in the code.

Reply:
👍